pkgs/tools/security/signify/default.nix

```diff
@@ -16,6 +16,10 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkg-config ];
   buildInputs = [ libbsd ];
 
+  preBuild = lib.optional (stdenv.hostPlatform != stdenv.buildPlatform) ''
+    substituteInPlace Makefile --replace 'shell pkg-config' 'shell ${stdenv.hostPlatform.config}-pkg-config'
+  '';
+
```

Comment:
```suggestion
  postPatch = ''
    substituteInPlace Makefile --replace "shell pkg-config" "shell $PKG_CONFIG"
  '';

```

the pkg-config setup hook adds the `$PKG_CONFIG` variable https://github.com/NixOS/nixpkgs/blob/244185c486428aecec8d9e5aa13d2ef6f6432935/pkgs/build-support/pkg-config-wrapper/setup-hook.sh#L26

when not cross-compiling its `pkg-config` when cross-compiling its `aarch64-unknown-linux-gnu-pkg-config`

Reply:
Github-workflow question (I use git a lot, github hardly ever): am I, as the PR submitter, the one who is supposed to act on these suggestions?  If so how?  I can't figure out how to make github emit a patch or commit-hash for them.  The webui will let me "add to batch" them, but then when I click "commit suggestions" nothing happens.

Just trying to figure out whose court the ball is in right now.

Reply:
make the changes locally
then make a commit containing them
then run `git rebase -i HEAD~2` and change the latest commit to `fixup` instead of `pick` 
then save and force push


if you want to apply the code review then clicking the "commit suggestion" button should show this
![image](https://user-images.githubusercontent.com/56650223/155264239-a3194133-56cd-4689-a073-ea0eddf1e7e2.png)

if you can apply it then git pull and fixup it into the first commit

Reply:
batch applying should show this 
![image](https://user-images.githubusercontent.com/56650223/155264506-7fa49f6b-26a2-4ba5-acd1-86f1e8a4081b.png)

Reply:
and yes github should add a way to get a patch from suggestions

Reply:
> make the changes locally

Okay, is there a better way to do this than painstakingly manually transcribing what I see in the web browser?  Like having github hand me a diff?  Doing it manually is really error-prone, especially for whitespace/formatting suggestions.

Thanks for the screenshots.  I had floating-bar-elements turned off.